src/V1Meta.js

```diff
@@ -54,4 +54,10 @@ export default class V1Meta {
         const headers = createHeaderObj(this.authHeader);
         return Promise.resolve(this.postFn(url, null, headers));
     }
+
+    queryDefinition(assetType) {
+        const url = `${this.urls.meta()}/${assetType}`;
+        const headers = createHeaderObj(this.authHeader);
+        return Promise.resolve(this.postFn(url, null, headers));
```

Comment:
If the meta definition should be a GET and not a POST, then a second getFn will need to be provided and not simply provide a get function as the POST (despite the name). It is named postFn because other methods require it to be a POST and not a GET. The question becomes, should querying meta definitions be a GET or a POST? A GET seems to make more sense, despite needing to pass in an additional getFn.